client/containers/AdminDashboard/AdminDashboard.tsx

```diff
@@ -1,58 +1,42 @@
 import React from 'react';
-
-import { GridWrapper } from 'components';
-import { apiFetch } from 'client/utils/apiFetch';
-
-import Chart from './Chart';
+import IframeResizer from 'iframe-resizer-react';
 
 require('./adminDashboard.scss');
 
-type State = any;
-
-class AdminDashboard extends React.Component<{}, State> {
-	constructor(props: {}) {
-		super(props);
-		this.state = {
-			pubPubData: {},
-			isLoading: true,
-		};
-	}
-
-	componentDidMount() {
-		apiFetch('/api/admin-dashboard', {
-			method: 'GET',
-		})
-			.then((data) => {
-				this.setState({
-					pubPubData: data,
-					isLoading: false,
-				});
-			})
-			.catch((err) => {
-				console.warn(err);
-			});
-	}
-
-	render() {
-		return (
-			<div id="dashboard-container">
-				<GridWrapper>
-					<h1>Admin Dashboard</h1>
-					{!this.state.isLoading && (
-						<div>
-							<Chart data={this.state.pubPubData.users} title="Users" />
-							<Chart data={this.state.pubPubData.communities} title="Communities" />
-							<Chart data={this.state.pubPubData.discussions} title="Discussions" />
-							<Chart
-								data={this.state.pubPubData.subscribers}
-								title="Newsletter Subscribers"
-							/>
-						</div>
-					)}
-				</GridWrapper>
-			</div>
-		);
-	}
-}
+type Props = {
+	impactData: any;
+};
+
+const AdminDashboard = (props: Props) => {
+	const { impactData } = props;
+	const { baseToken } = impactData;
+	const genUrl = (token) => {
```

Comment:
This could probably just be a constant instead of a function since it's called once and its argument is available right upstairs.